client/src/pages/Profile/ProfileDetail/BookingCard/BookingCard.tsx

```diff
@@ -0,0 +1,57 @@
+import { useState } from 'react';
+import Grid from '@mui/material/Grid';
+import Paper from '@material-ui/core/Paper';
+import Box from '@mui/material/Box';
+import { DateRange } from '@mui/lab/DateRangePicker';
+import { FormikHelpers } from 'formik';
+import BookingForm from './BookingForm/BookingForm';
+import { Sitter } from '../sampleData';
+import useStyles from './useStyles';
+
+interface Props {
+  sitter: Sitter;
+}
+
+const BookingCard = ({ sitter }: Props): JSX.Element => {
+  const classes = useStyles();
+
+  const [dateRange, setDateRange] = useState<DateRange<Date | null>>([null, null]);
+
+  const handleSubmit = (
```

Comment:
I see that these are just passed down to `BookingForm` without being used here. That's probably a good indication that we should move these down to that component.

Reply:
Applied